docs-ref-conceptual/Latest-version/includes/cli-install-linux-dnf.md

```diff
@@ -74,6 +74,11 @@ To install the Azure CLI, follow these steps:
    sudo dnf install azure-cli
    ```
 
+   > [!IMPORTANT]
+   > **RHEL 9.0 Compatibility**: Because Python 3.9 is end-of-life, Azure CLI 2.72 is the last
+   > version compatible with RHEL 9.0 systems using Python 3.9. Install Azure CLI 2.72 using: `sudo
```

Comment:
`using Python 3.9` is not accurate. It should be the OS does not have python3.12 package. In other word, less than RHEL 9.4.